sdk/helper/certutil/types.go

```diff
@@ -704,6 +704,45 @@ func (n NotAfterBehavior) String() string {
 	return "unknown"
 }
 
+type notBeforeBound int
+
+const (
+	ForbidNotBeforeBound notBeforeBound = iota
+	PermitNotBeforeBound
+)
+
+var notBeforeBoundNames = map[notBeforeBound]string{
+	ForbidNotBeforeBound: "forbid",
+	PermitNotBeforeBound: "permit",
+}
+
+func (n notBeforeBound) String() string {
+	if name, ok := notBeforeBoundNames[n]; ok && len(name) > 0 {
+		return name
+	}
+	return "unknown"
+}
+
+type NotAfterBound int
```

Comment:
Is there a reason why this is exported but `notBeforeBound` isn't? All of this could probably be private?

Reply:
No, you are right. Oversight from my part. Made it private.